README.md

```diff
@@ -27,13 +27,17 @@ Report issues/feature requests on [GitHub Issues](https://github.com/ossfriday/o
 
 ### Environment variables
 
-```bash
-bundle exec figaro install
+Create a config file for the environment variables:
+
+```
+cp config/application.example.yml config/application.yml
 ```
 
-You'll need to set at least `github_client_id` and `github_client_secret` environment variables. The values can be obtained by [registering a new GitHub OAuth application](https://github.com/settings/applications/new) with the Authorization callback URL pointing to your application with `/users/auth/github/callback` appended.
+Register a [new GitHub OAuth application](https://github.com/settings/applications/new) with the Authorization callback URL pointing to `http://localhost:3000/users/auth/github/callback`.
 
-Or for more information about using figaro, see https://github.com/laserlemon/figaro
+Update `config/application.yml`, setting the `github_client_id` and `github_client_secret` variables generated for your GitHub OAuth Application
```

Comment:
Anything in the setup which isn't marked optional reads as mandatory to me. Extra words just make it harder to scan. I'll have a second go at it.